### PR TITLE
Update Rate-reminder.md

### PR DESCRIPTION
**Pull request by TSkogemann**

### CODE REVIEW

File: docs/features/Rate-reminder.md [hunk 1]
```diff
@@ -36,3 +36,121 @@ You can find examples on how to implement **Rate Reminder** for the following pl
...
+# Version 2
+
+Ratereminders V2 has the following endpoints:
+
+ - [GET] Show
+ - [GET] Actions
+ - [Post] Action Events
+ - [Post] Answer
+
+# GUID
```
Comment (Casperhr) [inline docs/features/Rate-reminder.md:50]: We have a section here also
https://github.com/nstack-io/documentation/blob/master/docs/API.md#what-is-a-guid

maybe just point to that? and then extend it with what your wrote here, and maybe about what your mean by "storing your id" and what is a GUID

Comment (TSkogemann) [reply docs/features/Rate-reminder.md:50]: Good point. I will look into it now.

Comment (TSkogemann) [reply docs/features/Rate-reminder.md:50]: @Casperhr  i have linked to GUID. I'm not sure if i think that the guid part should be removed from rate-reminder. How do you like the current version ? :) 

I added that a user should not generate a new guid.

**Review by Casperhr — APPROVED** [review]

**Comment by Casperhr** [comment]
Please make a new PR where you update the section on GUID to be more detailed

**Pull request merged**